### PR TITLE
feat(signet-admin): #MED-302 admin signet page

### PR DESCRIPTION
**Pull request by niko-amosse**
## Describe your changes

display different folders of signet page

image

image

image

image

## Issue ticket number and link

[MED-302](https://jira.support-ent.fr/browse/MED-302)

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [ ] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [X] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: frontend/src/components/empty-state/empty-state.tsx [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
 
 interface EmptyStateProps {
-  imgSource?: string;
+  image?: string;
   title: string;
   description?: string;
 }
 
 export const EmptyState: React.FC<EmptyStateProps> = ({
-  imgSource,
```
Comment (MajorNovanir) [inline frontend/src/components/empty-state/empty-state.tsx:10]: pour les props conditionnelle, on préfèrera mettre une valeur par défaut ici.
```
 image = "empty-state.png",
 title,
 description,
```

File: frontend/src/components/empty-state/empty-state.tsx [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
+  image,
   title,
   description,
 }) => {
   const { t } = useTranslation();
   return (
     <div className="empty-state">
       <img
-        src={`/mediacentre/public/img/${imgSource || "empty-state.png"}`}
+        src={`/mediacentre/public/img/${image || "empty-state.png"}`}
```
Comment (MajorNovanir) [inline frontend/src/components/empty-state/empty-state.tsx:18]: et ducoup ici 
src={`/mediacentre/public/img/${image}`}

File: frontend/src/components/infinite-scroll-list/InfiniteScrollList.tsx [hunk 3]
```diff
@@ -91,10 +91,37 @@ export const InfiniteScrollList: React.FC<InfiniteScrollListProps> = ({
       }
       setIsLoading(false);
       setIsRemoveResource(false);
+    } else {
+      setIsLoading(true);
     }
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [allResourcesDisplayed, isRemoveResource]);
 
+  // useEffect(() => {
```
Comment (MajorNovanir) [inline frontend/src/components/infinite-scroll-list/InfiniteScrollList.tsx:100]: ?

File: frontend/src/hooks/useSignet.ts [hunk 4]
```diff
@@ -104,6 +110,38 @@ export const useSignet = () => {
...
+  }, [publicSignets, mySignets, user?.userId, favorites, pins]);
+
+  const mine = (signets: Signet[]) => {
+    return signets.filter(
+      (signet: Signet) => !signet.archived && signet.owner_id === user?.userId,
+    );
+  };
+
+  const shared = (signets: Signet[]) => {
+    console.log(signets);
```
Comment (MajorNovanir) [inline frontend/src/hooks/useSignet.ts:128]: log

**Review by MajorNovanir — APPROVED** [review]

File: frontend/src/components/infinite-scroll-list/InfiniteScrollList.tsx (hunk 3 above)
Comment (Mattox0) [reply frontend/src/components/infinite-scroll-list/InfiniteScrollList.tsx:100]: idk

**Pull request merged**